Tests/Common/PasswordManipulationTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Framework\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Zest\Common\PasswordManipulation;
+
+class PasswordManipulationTest extends TestCase
+{
+    public function generatePasswordTest()
+    {
+        $pass = new PasswordManipulation();
+        $this->assertNotEmpty($pass->generatePassword(11));
```

Comment:
It should add more assertions here.

For example, assert the password length and password pattern validation.

Reply:
It should add `test-based` prefix to name method name. And PHPUnit will know this methods are about tests.

For example, naming the `testGeneratePassword`.

Reply:
> password

Password pattern is not need because isValid method do that already